app/view.py

```diff
@@ -62,6 +67,27 @@ def plot(self, params=None, layout_dict: dict = None, show: bool = False):
             fig.show()
         return fig
 
+    def real_seir_plot(self, parameters):
```

Comment:
Sollte in plot() integriert werden

Reply:
Ist es nicht sinnvoller die Plots zu trennen, anstatt mit if Anweisungen die Daten auseinander zu halten?
Der Plot ist extra so gemacht, dass "S" noch Anfällige die rechte y Achse gezeigt wird, da sonst ein zu großer unterschied besteht

Reply:
Naja ist meiner Meinung nach nicht sehr intuitiv nutzbar. Grundsätzlich sollten die Funktionen plot() und real_seir_plot() ja das Gleiche, nur mit anderen Daten, tun. Die if Anweisung ist tatsächlich nicht sinnvoll, plot() sollte einfach nur mit den gewünschten Werten aufgerufen werden. 

Welche Probleme treten denn auf wenn du die realen Daten in plot() reinwirfst? Die rechte y Achse zu integrieren scheint mir bis jetzt das Einzige zu sein über das man sich Gedanken machen müsste.

Reply:
Ok, habs in Plot eingebaut,

die unteren Lösch ich raus

from plotly.graph_objs import Line
from plotly.graph_objs.scatter.marker import Line

wenn ich es zeitlich schaff, lad ich es morgen hoch

Reply:
Super, du kannst die neuen Commits direkt in diesem Branch hochladen, also kein neues PR